static/app/components/charts/chartWidgetLoader.tsx

```diff
@@ -0,0 +1,56 @@
+import {captureException} from '@sentry/core';
+import {useQuery} from '@tanstack/react-query';
+
+import Placeholder from 'sentry/components/placeholder';
+import {t} from 'sentry/locale';
+import type {LoadableChartWidgetProps} from 'sentry/views/insights/common/components/widgets/types';
+
+class WidgetImportError extends Error {
+  constructor(options?: ErrorOptions) {
+    const message =
+      'Unable to import widget: widget file not found or widget not exported as default export.';
+    super(message, options);
+    this.name = 'WidgetImportError';
+  }
+}
```

Comment:
Just curious, what's the advantage of extending the `Error` class like this? Fingerprinting?

Reply:
ah! probably not much advantage here, previously had another error and I prefer using the error instance for control flow (vs msg). (e.g. `... catch(err) { if (err instanceof WidgetImportError) ... }` -- Will clean this up